core/driver/sources/src/main/scala/org/virtuslab/ideprobe/IdeProbePaths.scala

```diff
@@ -17,8 +17,8 @@ case class IdeProbePaths(
 
 object IdeProbePaths {
   val Default: IdeProbePaths = {
-    // TODO: replace None parameters with loading defaults from reference.conf
-    from(PathsConfig(None, None, None, None, None, None, None))
+    val config = IntelliJFixture.readIdeProbeConfig(Config.fromReferenceConf, "probe")
```

Comment:
we shouldn't need to repeat this "probe" anywhere. 
Also `IntelliJFixture.readIdeProbeConfig(Config.fromReferenceConf, "probe")` could be stored as a lazy val to avoid reading it multiple times.